View Controllers/TimelineViewController.m

```diff
@@ -111,20 +112,21 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     
     //retweets
     UIImage *retweeticon = [UIImage imageNamed:@"retweet-icon"];
-    if(tweet.retweeted) {
```

Comment:
Careful about what you check in. You should be un-staging all changes then reading and staging each line, one at a time, before committing.